sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/AuthController.java

```diff
@@ -77,4 +82,16 @@
         request.getSession().invalidate();
         return Result.ofSuccess(null);
     }
+
+    @RequestMapping(value = "/check")
+    public Result<?> check(HttpServletRequest request) {
+        AuthService.AuthUser authUser = authService.getAuthUser(request);
+        if (authUser == null) {
+            return Result.ofFail(-1, "Not logged in");
+        }
+        if (authService instanceof FakeAuthServiceImpl || request.getSession().getAttribute(SimpleWebAuthServiceImpl.WEB_SESSION_KEY) == null) {
+            request.getSession().setAttribute(SimpleWebAuthServiceImpl.WEB_SESSION_KEY, authUser);
```

Comment:
I think this judgement can be removed, since if use `FakeAuthServiceImpl`,  no need to create the backend session.
If use `SimpleWebAuthServiceImpl`,  the backend session should be created when login, not in the check method,
and the backend session has its own expire time.

Reply:
Very right, after delete this code, I didn't find any problems when `session`
 has no `WEB_SESSION_KEY`.